bundles/org.openhab.binding.solarman/src/main/resources/OH-INF/i18n/solarman.properties

```diff
@@ -0,0 +1,103 @@
+# add-on
+
+addon.solarman.name = Solarman Logger Binding
+addon.solarman.description = This is the binding for Solarman Logger
+
+# thing types
+
+thing-type.solarman.logger.label = Solarman Binding Thing
+thing-type.solarman.logger.description = Solarman Logger Binding Thing
+
+# thing types config
+
+thing-type.config.solarman.logger.additionalRequests.label = Additional Requests
+thing-type.config.solarman.logger.additionalRequests.description = Additional requests besides the ones defined in the inverter definition. Format is mb_functioncode1:start1-end1, mb_functioncode2:start2-end2,... Example 0x03:0x0000-0x0100,0x03:0x0200-0x0300
+thing-type.config.solarman.logger.hostname.label = Logger hostname
```

Comment:
This file seems out of date when comparing to thing-type.xml so fix this by re-running the i18n plugin